### ISSUE TITLE
Transcript

### ISSUE THREAD

**Issue by Oliver-Malkin** [j4cobgarby/jailbreak#1]
Would be fun to have a transcript at the end of the game or when you want to

**Comment by j4cobgarby** [comment]
Yeah true, in fact we were saying this during the hackathon, it would be really nice cause then you can prove that you've beaten the warden lol
Shouldn't be too hard to implement, I can have a go at this after fixing a bunch of small bugs, unless you wanna give it a go?
